projects/sound-open-firmware/Dockerfile

```diff
@@ -20,39 +20,52 @@
 
 FROM gcr.io/oss-fuzz-base/base-builder
 
-RUN apt-get -y update && \
-	    apt-get install -y \
-	    	autoconf \
-		bison \
-		build-essential \
-		flex \
-		gawk \
-		gettext \
-		git \
-		gperf \
-		help2man \
-		libncurses5-dev \
-		libssl-dev \
-		libtool \
-		libtool-bin \
-		pkg-config \
-		software-properties-common \
-		texinfo \
-		udev \
-		cmake \
-		libglib2.0-dev
-
-ARG CLONE_DEFAULTS="--depth 5"
-
-# Use ToT alsa utils for the latest topology patches.
-RUN cd /tmp && \
-git clone $CLONE_DEFAULTS https://github.com/thesofproject/alsa-lib.git && \
-git clone $CLONE_DEFAULTS https://github.com/thesofproject/alsa-utils.git && \
-cd /tmp/alsa-lib && ./gitcompile &&  make install && \
-cd /tmp/alsa-utils && ./gitcompile &&  make install
-
-ARG GITHUB_SOF=https://github.com/thesofproject
-
-RUN cd $SRC && git clone $CLONE_DEFAULTS $GITHUB_SOF/sof
-WORKDIR sof
+# Base packages
+RUN apt-get -y update
+RUN apt-get install -y \
+      gettext git libc6-dev-i386 libglib2.0-dev libncurses5-dev \
+      libtool ninja-build python3-pip wget
+RUN pip3 install west
+
+WORKDIR $SRC
+
+# Zephyr SDK: (FIXME: there is a much smaller variant with only host tools...)
+RUN wget -q https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.16.0/zephyr-sdk-0.16.0_linux-x86_64.tar.xz
```

Comment:
any reason we are tied to a particular release?

Reply:
Just a general intuition that Zephyr SDK updates tend to be high churn and need some manual attention anyway.  Also as I understand the setup, docker isn't going to be sensitive to things like remotely-pulled "latest" links and we'd need to manually rebuild the image anyway if the SDK changed underneath it.

Reply:
> Just a general intuition that Zephyr SDK updates tend to be high churn and need some manual attention anyway.

Confirmed. SDK flag day learned the hard way this week: https://github.com/thesofproject/sof/issues/7614

Reply:
Is there a way would could "inject" the version into this script? Updating things in this repo at any level of regularity is not simple.

Reply:
https://docs.docker.com/engine/reference/builder/#arg ?

It could also be a more dynamic ENV that defaults to 0.16.1 (why 0.16.0 BTW?)

Reply:
Uh... because that line of the script predates the .1 release and I missed it? :)

I'll dig up a latest link (I'm like 80% sure we have one) and use that.  It won't hurt anything, and if nothing else it's a good opportunity to figure out which SDK subsets we really need so the image doesn't have to pull all the architecture toolchains we don't care about.